src/amo/components/CollectionManager/index.js

```diff
@@ -128,26 +132,26 @@ export class CollectionManagerBase extends React.Component<Props, State> {
       user: collection.authorUsername,
       slug,
     }));
-  }
+  };
 
   onTextInput = (
     event: ElementEvent<HTMLInputElement | HTMLTextAreaElement>
   ) => {
     event.preventDefault();
     this.setState({ [event.target.name]: event.target.value });
-  }
+  };
 
   onSearchAddon = (filters: SearchFilters) => {
     // TODO: implement onSearchAddon
     // https://github.com/mozilla/addons-frontend/issues/4590
     log.debug('TODO: handle seaching for add-on', filters);
-  }
+  };
 
   onAddonSelected = (suggestion: SuggestionType) => {
     // TODO: implement onAddonSelected
     // https://github.com/mozilla/addons-frontend/issues/4590
     log.debug('TODO: handle selecting an add-on', suggestion);
-  }
+  };
```

Comment:
Huh, these are needed now? Weird, I thought they were a linting error. (Unneeded semicolons.)

Reply:
Seems like not though. 🤷‍♂️

Reply:
They are neither needed nor a linting error. It would have been a linting error if this were Firefox code, but I guess our configuration allows both to include and exclude them. I find having them excluded annoying, as it seems inconsistent to me, which is why I added them.